config/routes.rb

```diff
@@ -10,6 +10,9 @@
   get '/dashboard', to: 'users#show'
   resources :carts, only: [:create]
 
+  resources :users, only: :show do
+    resources :orders, only: :index
+  end
```

Comment:
Looking back at the user story for this one it says the path should be slash orders, but IIRC this will actually make the path '/users/:id/orders'.